workshops/PREFACE.md

```diff
@@ -4,23 +4,29 @@ _Adapted from [here](https://medium.freecodecamp.com/things-i-wish-someone-had-t
 
 ---
 
-If you're reading this, that means that you've decided to embark on the journey of learning to code. Congratulations and welcome, we're excited to have you.
+If you're reading this, that means that you've decided to embark on the journey of learning to code. Congratulations and welcome, we're excited for you. Before you get started, there are 3 things we think you should know:
 
-## Coding is Not Special
+[1. Coding is Not Special](#1-coding-is-not-special)  
+[2. It Never Works the First Time](#2-it-never-works-the-first-time)  
+[3. Persistence Beats Method](#3-persistence-beats-method)
 
-First and foremost, coding is a skill like any other. Like learning a language, there's grammar and vocabulary to acquire. Like math, there are processes to work through specific types of problems. Like all kinds of craftsmanship and art-making, there are techniques and tools and best practices that people have developed over time, specialized to different tasks, that you're free to use or modify or discard.
+## 1. Coding is Not Special
 
-Someone will always tell you you're doing it wrong. There are almost always many different approaches to a particular problem with no single "right way". A lot of programmers get very good at advocating for their preferred way, but that doesn't mean it's the One True Path.
+First and foremost, coding is a skill like any other. Like learning a language, there's grammar and vocabulary to acquire. Like math, there are processes to work through specific types of problems.
 
-## It Never Works the First Time
+Like any type of craftsmanship and art-making, there are techniques and best practices that have developed over time.
+
+Someone will always tell you you're doing it wrong. Coding problems generally have more than one solution. A lot of programmers get good at advocating for their personal preferences, but keep in mind there's one way to bake a cake.
+
+## 2. It Never Works the First Time
 
 When you first start learning to code, you'll very quickly run up against this particular experience: you think you've set up everything the way you're supposed to, you've checked and re-checked it, and it still. doesn't. work. You don't have a clue where to begin trying to fix it, and the error message (if you're lucky enough to have one at all) might as well say "fuck you". You might be tempted to give up at this point, thinking that you'll never figure it out, that you're not cut out for this.
```

Comment:
We're probably not trying to be family friendly, but I think that it might not be a stellar idea to have "fuck you" in our preface